staging/src/k8s.io/kubectl/pkg/drain/drain.go

```diff
@@ -65,6 +65,10 @@ type Helper struct {
 	// won't drain otherwise
 	SkipWaitForDeleteTimeoutSeconds int
 
+	// AdditionalFilters are applied sequentially after base drain filters to
+	// exclude pods using custom logic.
+	AdditionalFilters []PodFilter
```

Comment:
Do we need multiple?  Or is this just reusing other code?

Reply:
Definitely you could get by with one instead of more than one if you're writing the filter yourself.  I think I prefer it as a slice of filters rather than just one filter.

Reply:
If you leave it as an array, document the behavior (short circuit).